### DISCUSSION TITLE
How upload Mp3 File?

### DISCUSSION THREAD

**Discussion by Ichsan-06** [uasoft-indonesia/badaso#1057] [Q&A]
can i upload mp3 file ?? because when iam trying to upload file .mp3 error
![image](https://github.com/uasoft-indonesia/badaso/assets/71815655/33439391-7b4e-4214-97a3-69fd499f2a9e)

**Comment by LemonSync** [comment]
First, make sure it's an mp3 file. Second, you can use the scroll file method or upload (select) it manually. And done, if it still doesn't work, just upload it using the terminal
